File: CSharpVer/CSharpVer/FileUtil.cs
using System;
using System.IO;

namespace CSharpVer
{
    class FileUtil
    {
        private static string GetPath(string file)
        {
            var enviroment = Environment.CurrentDirectory;
            string projectDirectory = Directory.GetParent(enviroment).Parent.FullName;

            return Path.Combine(projectDirectory, file);
        }
        public static void WriteToFile(string file, byte[] fileContent)
        {
            Console.WriteLine("Writing bytes to file...");
            File.WriteAllBytes(GetPath(file), fileContent);
            Console.WriteLine("Done writing bytes to file...");
        }

        public static byte[] ReadBytesFromFile(string file)
        {
            Console.WriteLine("Reading bytes from file...");
            byte[] data = File.ReadAllBytes(GetPath(file));
            Console.WriteLine("Done reading from file...");
            return data;
        }

        public static byte[] combineBytes(byte[] a, byte[] b)
        {
            byte[] combined = new byte[a.Length + b.Length];
            Array.Copy(a, 0, combined, 0, a.Length);
            Array.Copy(b, 0, combined, a.Length, b.Length);

            return combined;
        }
    }
}

File: JavaVer/src/FileUtil.java
import java.io.File;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;

public class FileUtil {
    public static void writeToFile(File file, byte[] fileContent) {
        System.out.println("Writing bytes to file...");
        Path path = Paths.get(file.getAbsolutePath());
        try {
            Files.write(path, fileContent);
        } catch (IOException ex) {
            ex.printStackTrace();
        }
        System.out.println("Done writing bytes to file...");
    }

    public static byte[] readBytesFromFile(File file) {
        System.out.println("Reading bytes from file...");
        Path path = Paths.get(file.getAbsolutePath());
        byte[] data = new byte[0];
        try {
            data = Files.readAllBytes(path);
        } catch (IOException ex) {
            ex.printStackTrace();
        }
        System.out.println("Done reading from file...");
        return data;
    }

    public static byte[] combineBytes(byte[] a, byte[] b){
        byte[] combined = new byte[a.length + b.length];
        System.arraycopy(a, 0, combined, 0, a.length);
        System.arraycopy(b, 0, combined, a.length, b.length);

        return combined;
    }
}
